lib/fake_braintree/customer.rb

```diff
@@ -118,6 +122,11 @@ def set_default_credit_card(credit_card_hash)
 
     def generate_credit_cards_from(new_credit_card_hash)
       if new_credit_card_hash.present? && new_credit_card_hash.is_a?(Hash)
+        load_payment_method!(
+          new_credit_card_hash.delete('payment_method_nonce'),
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.